package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "eslint-config-skyscanner",
-  "version": "5.4.0",
+  "version": "5.4.1",
```

Comment:
Can you make this `6.0.0` as this will be a breaking change by us removing this plugin

Reply:
You're right, it should be breaking. I've updated it.